bot.rb

```diff
@@ -355,6 +355,7 @@ def process_code_signing(issue)
     def last_responding_user(issue)
       client.issue_comments(SLUG, issue.number)
       link_to_last_page = client.last_response.rels[:last]
+      return unless link_to_last_page
```

Comment:
Related: https://developer.github.com/v3/guides/traversing-with-pagination/

Also, looking at how this function is used, returning nil here would result in a cycle in these issues where it returns `nil` again: 

1. After `ISSUE_WARNING` months of no activity the bot attaches the `AWAITING_REPLY` label in the else branch on line 310
2. `ISSUE_CLOSED` months later the bot takes the true branch on line 294
3. In a case where this now returns nil it will erroneously take the else branch on line 304 and remove the `AWAITING_REPLY` 
4. `goto 1`

Reply:
From the docs it sounds like these should always exist, which they don't. Quite ironically, I haven't found a way to report issues about the GitHub API documentation as it's not Open Source anymore. Would you happen to know how anything about that @ohayon?

Reply:
Not sure about reporting issues for the API @milch, this is interesting. Thanks for finding that new issue! Do you know what we can do to fix it here?

Reply:
Somehow I stumbled upon [this issue](https://github.com/octokit/octokit.rb/issues/711) where one of the members said that he will "will file a bug internally with the API team". So maybe opening an issue there, might be reported by the members to the GitHub team, if I'm not mistaking this. IIRC, GitHub even mentioned `octokit`, so maybe it's an officially supported library.

Reply:
@ohayon Easiest fix would probably be to introduce a variable on line 356 to capture the issue comments, and then return the user from there. Something like 

```ruby
first_page = client.issue_comments(SLUG, issue.number)
link_to_last_page = client.last_response.rels[:last]
last_page = link_to_last_page.get.data if link_to_last_page
return (last_page || first_page).last.user.login
```